src/networking/index.ts

```diff
@@ -49,16 +49,16 @@ export default class Client {
     this.timeout = timeout;
   }
 
-  post(path: string, body: unknown) {
-    return this.request('POST', this.url(path), body);
+  post<TData = unknown, TBody = unknown>(path: string, body: TBody) {
+    return this.request<TData, TBody>('POST', this.url(path), body);
   }
 
-  patch(path: string, body: unknown) {
-    return this.request('PATCH', this.url(path), body);
+  patch<TData = unknown, TBody = unknown>(path: string, body: TBody) {
+    return this.request<TData, TBody>('PATCH', this.url(path), body);
   }
 
-  get(path: string, query?: unknown) {
-    return this.request('GET', this.url(path, query));
+  get<TData = unknown>(path: string, query?: unknown) {
+    return this.request<TData>('GET', this.url(path, query));
```

Comment:
Ensures to pass the Response type, and payload where needed to improve type safety.